firebase-vscode/webviews/EmulatorPanel.tsx

```diff
@@ -33,14 +33,20 @@ const DEFAULT_EMULATOR_UI_SELECTIONS: EmulatorUiSelections = {
  */
 export function EmulatorPanel({
```

Comment:
nit: I just noticed that this is at the top level of `webviews/`. Can you put it inside `components/` with the other components?

Reply:
Done.